coalib/results/result_actions/ShowAppliedPatchesAction.py

```diff
@@ -32,6 +26,7 @@ def apply(self,
         FILE_DIFF_DICT_INDEX = 2
         SECTION_INDEX = 3
 
+        from coalib.output.ConsoleInteraction import format_lines
```

Comment:
The `import` statement had to be placed here instead of the usual - top of the file - because coala was encountering an error with circularly dependent import.

Reply:
Then put the function in a different module which doesnt cause a circular dependency

Reply:
There's a circular dependency problem with `ShowAppliedPatches.py` regardless of whether I put the function in `ConsoleInteraction.py` or `coala_main.py`. Now, the function `format_lines()` is used only in these 3 modules-- `ShowAppliedPatches.py`, `ConsoleInteraction.py` and `coala_main.py`. 

Therefore, I IMHO I think that we need to put the function in one of these files only. Also, I don't think that it should be inside `ShowAppliedPatches.py`. 

Hence, the problem. :confused: 

Please suggest a way out of it @jayvdb

Reply:
It is your task to find the solution.
I already know it.
And telling it to you doesn't help you learn.